src/relayFeeCalculator/relayFeeCalculator.ts

```diff
@@ -18,7 +18,11 @@ import { Deposit } from "../interfaces";
 
 // This needs to be implemented for every chain and passed into RelayFeeCalculator
 export interface QueryInterface {
-  getGasCosts: (deposit: Deposit, amountToRelay: BigNumberish, relayerAddress: string) => Promise<BigNumberish>;
+  getGasCosts: (
+    deposit: Deposit,
+    amountToRelay: BigNumberish,
+    relayerAddress: string
+  ) => Promise<{ gasCost: BigNumberish; gasTokenCost: BigNumberish }>;
```

Comment:
Since we use this in two+ places, we should place this in a type

Reply:
Applied: https://github.com/across-protocol/sdk-v2/pull/453/commits/4d6c8ad2cea64444f9d4642003fa7bf395550723